guacamole/src/main/frontend/src/app/auth/service/authenticationService.js

```diff
@@ -252,6 +252,24 @@ angular.module('auth').factory('authenticationService', ['$injector',
 
     };
 
+    /**
+     * Makes a request to revoke an authentication token using the token REST
+     * API endpoint, returning a promise succeeds only if the token was
```

Comment:
_that_ succeeds?

Reply:
Oops.

Reply:
There's a second occurrence of this in the documentation of the function that this was copied from. I've corrected both in a fresh commit.